integration_tests/README.md

```diff
@@ -378,6 +378,15 @@ As an example, here is the `spark-submit` command with the cudf_udf parameter on
 $SPARK_HOME/bin/spark-submit --jars "rapids-4-spark_2.12-22.06.0-SNAPSHOT.jar,cudf-22.06.0-SNAPSHOT-cuda11.jar,rapids-4-spark-tests_2.12-22.06.0-SNAPSHOT.jar" --conf spark.rapids.memory.gpu.allocFraction=0.3 --conf spark.rapids.python.memory.gpu.allocFraction=0.3 --conf spark.rapids.python.concurrentPythonWorkers=2 --py-files "rapids-4-spark_2.12-22.06.0-SNAPSHOT.jar" --conf spark.executorEnv.PYTHONPATH="rapids-4-spark_2.12-22.06.0-SNAPSHOT.jar" ./runtests.py --cudf_udf
 ```
 
+### Enabling fuzz tests
+
+Fuzz tests are intended to find more corner cases in testing. We disable them by default because they might randomly fail. 
+The tests can be enabled by appending th option `--fuzz_test` to the command.
+
+   * `--fuzz_test` (enable the fuzz tests when provided, and remove this option if you want to disable the tests)
+
+To reproduce an error appearing in the fuzz tests, you also need to add the flag `--debug_tmp_path` to save the test data.
```

Comment:
Could we combine these two together? So if you pass in `--fuzz_test_debug_path` or something like that the tests are enabled? To be clear I am fine with keeping the debug_tmp_path as it is. If this is really hard you don't need to do it.